### ISSUE TITLE
Integrate task anonymization script into GTG

### ISSUE THREAD

**Issue by Neui** [getting-things-gnome/gtg#735]
Idea: If someone has a problem that might require their tasks file, it would be nice if they could just invoke GTG it some way (command line? Maybe also via the GUI?) which would then anonymize the tasks file and output that via something (file chooser? stdout to redirect? Also consider flatpaks sandbox) so it can be send over without having to get and invoke the script manually.

**Comment by diegogangl** [comment]
Heh I hadn't seen this email, but I commented the same thing in #733
I think we should include it as an option in an export plugin. We could offer an XML export (esentially copy gtg_data.xml to another location), and have anynomization as an option there

**Comment by Neui** [comment]
I'm ok with the export plugin (although I am a bit unsure whenever this would be easy to find...), but I'd like to at least make it available for CLI in case the GUI doesn't work or something. I guess we could just redirect/document externally where this functionality is.

**Comment by diegogangl** [comment]
Yeah definitely, we can have a separate module that also runs as a script

**Comment by pentux-GitHub** [comment]
Coming back to this issue with a slightly different angle, after hitting related problems while working on the 0.7 data layer (#1289: the loader currently rejects an entire real-world 0.6 file because of a single non-UUID task id).
The underlying need here seems to be "share something that lets maintainers reproduce a data-related bug", and anonymization is only one way to get there. There might be a lighter path: make the loader keep a small structural log of what it could not read. Something like task #47: skipped, invalid id (expected UUID). Structure only, no user content, so there is nothing to anonymize by construction. The user can read the whole thing and paste it into an issue as-is.
This would also act as a safety net for the 0.6 → 0.7 migration: instead of a silent crash on unexpected data, users would get a readable trace, and unknown data issues would surface as reproducible fragments.
Would this direction be of interest? If so, I could open a small PR to discuss on concrete code, once #1265 has landed.
